src/renderer/features/multisigs/model/multisigs-model.ts

```diff
@@ -15,69 +12,71 @@ import {
   ExternalType,
   SigningType,
   WalletType,
+  Wallet,
 } from '@shared/core';
 import { networkModel, networkUtils } from '@entities/network';
 import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
 import { MultisigResult, multisigService } from '@entities/multisig';
-import { multisigUtils } from '../lib/mulitisigs-utils';
 import { isEthereumAccountId, toAddress } from '@shared/lib/utils';
 import { CreateParams } from '@entities/wallet/model/wallet-model';
 import { notificationModel } from '@entities/notification';
+import { multisigUtils } from '../lib/mulitisigs-utils';
+
+const MULTISIG_DISCOVERY_TIMEOUT = 30000;
 
 const multisigsDiscoveryStarted = createEvent();
-const chainConnected = createEvent<ChainId>();
+const multisigSaved = createEvent<GetMultisigsResult>();
 
-type GetMultisigsParams = {
-  chain: Chain;
-  accounts: Account[];
-};
+const $chainsSupportingMultisigDiscovery = combine(networkModel.$chains, (chains) =>
```

Comment:
I am doing TDD with this refactor, and I'm facing an issue:
> thrown: "Exceeded timeout of 5000 ms for a test.

Reducing the `MULTISIG_DISCOVERY_TIMEOUT` doesn't help, I checked and the interval is ricking as expecting, however, the issue seems to be with this combine, the `chainsSupportingMultisigDiscovery` is always `[]` and this functions to filter the chains is never called. Any hint on what is needed to trigger this?

Reply:
Take a look at `balance-sub-model.test.ts` - `await jest.runAllTimersAsync();`
When you encounter a timer, jest needs to run it so the timeout is not raised

Reply:
Thanks, this is not linked to the timer though I think. I added this to the tests, and they're still failing. The timer is ticking however this store `networkModel.$chains` is never getting changed it seems.. and the sample containing `clock: [multisigDiscoveryTriggered, $chainsSupportingMultisigDiscovery],` doesn't get trigger, not because of `multisigDiscoveryTriggered` but because of `$chainsSupportingMultisigDiscovery`. We used to call `$networkModel.$chains` with `once`, but:
- I don't know if that's actually the issue
- we can't use `once` in a combine it seems